LICENSE.txt

```diff
@@ -9,7 +9,7 @@ software.
 
 1. Definitions
 
-The terms "reproduce," "reproduction," "derivative works," and "distribution"
+The terms "reproduce", "reproduction", "derivative works" and "distribution"
```

Comment:
No clue...  my first thought is that the risk of the license being wrong isn't worth it.